assignments/events/2. backgroundChange/main.js

```diff
@@ -0,0 +1,5 @@
+var btn = document.querySelector("button");
+btn.addEventListener("click", function() {
+  var color = "#" + ((Math.random() * 0xffffff) << 0).toString(16);
```

Comment:
In `((Math.random() * 0xffffff) << 0).toString(16)` what `<<` does ?

Reply:
If you need to understand `<<` go through this article https://hackernoon.com/programming-with-js-bitwise-operations-393eb0745dc4